### ISSUE TITLE
Unable to resize font in archinstall

### ISSUE THREAD

**Issue by Torxed** [archlinux/archinstall#1509]
A user noted that while setting font size in arch releng iso, it doesn't translate into `archinstall`.
This is probably because we loaded a specific font with a specific font size to cover cyrillic issues etc.

https://discord.com/channels/726808923662712894/726809631342592081/1026892316268560404

**Comment by svartkanin** [comment]
Yes that's right, it's due to the custom font setting :(

**Comment by codefiles** [comment]
Suppose a conditional was added to the `else` statement in the code below to make it an `elif` that checked if the current font was the default ISO font. That way if the user set a font before `archinstall` it would not match the default ISO font and `_set_font()` would not run.

https://github.com/archlinux/archinstall/blob/5c8eb7144d9583a8a117a4e892b1e78be1324a28/archinstall/lib/translationhandler.py#L53-L56

**Comment by svartkanin** [comment]
The problem here is that to be able to display a fair amount of languages correctly in the language change menu we need to set a font that supports cyrillic.

I see some options here:
- if custom font was set outside archinstall, don't override it and just break some language displays
- stop displaying languages in their own translation and revert back to displaying all in English and only set a font if the language was selected

**Comment by codefiles** [comment]
> The problem here is that to be able to display a fair amount of languages correctly in the language change menu we need to set a font that supports cyrillic.
> 
> I see some options here:
> 
>     * if custom font was set outside archinstall, don't override it and just break some language displays
> 
>     * stop displaying languages in their own translation and revert back to displaying all in English and only set a font if the language was selected
@svartkanin, could another option be to change the console font temporarily when displaying the language menu?

In my suggestion I didn't consider if a user sets a console font they may do it without checking/knowing if the font supports the language they plan to use. I was going by if they manually set it that is what they want to use.

**Comment by svartkanin** [comment]
Yeah that would work as well actually, sorry i read over your comment too quickly :)

**Comment by svartkanin** [comment]
@codefiles actually I'm not entirely sure we can detect if a custom font was set. Last time I tried I wasn't able to retrieve the currently set font

**Comment by codefiles** [comment]
I see what you mean.

**Comment by svartkanin** [comment]
I gave this another go, but I wasn't able to determine the currently set font. 
In that case, i'd vote for reverting back to displaying all languages in English until one is selected. If the selected language has characters that are outside the default font support, we'll load another one. This way only a smaller subset of languages will potentially overwrite the custom setting.
